source/agora/utils/Log.d

```diff
@@ -48,6 +52,58 @@ else
     /// Initialize the logger
     static this ()
     {
-        Log.defaultConfig();
+        auto appender = new AppendConsole();
```

Comment:
Might want to update [this one too](https://github.com/bpfkorea/agora/blob/3df33d16faf60d682e8312c0f5b3702307e2a4b8/source/agora/common/TransactionPool.d#L49)?

Reply:
Why is that ?

Reply:
Ah sorry I misread the diff, I thought this was inside the mixin template.